lib/rspec/rails/view_rendering.rb

```diff
@@ -6,6 +6,8 @@ module Rails
     module ViewRendering
       extend ActiveSupport::Concern
 
+      # @!attribute [r]
```

Comment:
Oops, that was a copy-paste mistake. But yes, I believe you are correct, we should note that `controller=` is a `private` API call. Thanks for catching it!